### PR TITLE
Added Api#getCollection().

### PR DESCRIPTION
**Pull request by n1k0**
r=? @almet

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -311,6 +311,25 @@ export default class KintoApi {
...
 
   /**
+   * Retrieves information for a given collection.
+   *
+   * @param  {String} id      The collection name.
+   * @param  {Object} options The options object.
+   * @return {Promise<{Object}, Error>}
+   */
+  getCollection(id, options={}) {
+    const { bucket, headers } = {
```
Comment (almet) [inline src/index.js]: Not sure why this is better than doing it on two different lines?

**Comment by almet** [comment]
r+!

File: src/index.js (hunk 1 above)
Comment (n1k0) [inline src/index.js]: How would you write it on two lines?

Comment (almet) [inline src/index.js]:
```js
const bucket = this.defaultBucket;
const headers = {};
```
?

Comment (almet) [inline src/index.js]:
```
const bucket = this.defaultBucket;
const headers = {};
```
?

Comment (n1k0) [inline src/index.js]: Sigh, now I realize I'm not merging the options anymore here as in other methods. I really need to homogenize options processing.

**Comment by almet** [comment]
Great! Can you issue a 0.2.3 (or tell me how to do it?)

**Comment by n1k0** [comment]
You're a npm contributor for the package so you should be able to release it.

Basically that's:

 ```
$ git co master
$ git pull
Edit package.json and bump the version to 0.2.3 
$ git ci -a -m"Bump v0.2.3"
$ git push origin master
$ git tag v0.2.3
$ git push origin v0.2.3
$ npm run build
$ npm run dist
$ npm login
$ npm publish
```

**Comment by almet** [comment]
hmm, when doing `npm run build` I've got the following errors:

```
You have mistakenly installed the `babel` package, which is a no-op in Babel 6.
Babel's CLI commands have been moved from the `babel` package to the `babel-cli` package.

    npm uninstall babel
    npm install babel-cli

See http://babeljs.io/docs/usage/cli/ for setup instructions.
```

Investigating.

**Pull request merged**